migrations/20161104170802-sample-remove-name-index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    queryInterface.removeIndex('Samples', 'SampleName');
+  },
+
+  down: function (queryInterface, Sequelize) {
+    queryInterface.addIndex(
+      'Samples',
+      ['name'],
+      {
```

Comment:
Hmm.. I see your point. I can add the index back in if @shriramshankar has no issues.